### PR TITLE
Update functional test documentation

### PR DESCRIPTION
**Pull request by AaronFeickert**
## Description

Updates the functional test documentation to fix issues and improve readability.

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature/Enhancement (non-breaking change which adds functionality or enhances an existing one)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update
- [ ] Refactor
- [ ] New or updated tests
- [ ] Dependency Update

## Notes to Reviewers

These directions were tested on a macOS 15.1.1 installation.

## Checklist

- [x] I have performed a self-review of my code.
- [x] I have commented my code where necessary.
- [x] I have updated the documentation if needed.
- [x] My changes do not introduce new warnings.
- [x] I have added tests that prove my changes are effective or that my feature works.
- [x] New and existing tests pass with my changes.

## Related Issues

None.

### CODE REVIEW

**Review by delbonis — APPROVED** [review]

File: functional-tests/README.md [hunk 1]
```diff
@@ -48,22 +50,29 @@ pipx install poetry
 brew install poetry
 ```
 
-Check, that the `poetry` is installed:
+Check, that `poetry` is installed:
 ```bash
 poetry --version
 ```
 
 Finally, install all test dependencies by running:
```
Comment (storopoli) [inline functional-tests/README.md:58]:
```suggestion
Finally, install all test dependencies by running:

```

File: functional-tests/README.md [hunk 2]
```diff
@@ -48,22 +50,29 @@ pipx install poetry
 brew install poetry
 ```
 
-Check, that the `poetry` is installed:
+Check, that `poetry` is installed:
```
Comment (storopoli) [inline functional-tests/README.md:53]:
```suggestion
Check, that `poetry` is installed:

```

File: functional-tests/README.md [hunk 3]
```diff
@@ -48,22 +50,29 @@ pipx install poetry
...
 ```bash
 poetry --version
 ```
 
 Finally, install all test dependencies by running:
 ```bash
-poetry install
+poetry install --no-root
```
Comment (storopoli) [inline functional-tests/README.md:60]: It is worth adding that the `--no-root` is

> `Do not install the root package (the current project).`

**Review by storopoli — APPROVED** [review]
ACK bc73b2fd26b22df44b8dc3a522bd8b92b395429a

**Pull request merged**